addons/stock/models/stock_warehouse.py

```diff
@@ -346,6 +346,9 @@ def _create_or_update_sequences_and_picking_types(self):
             else:
                 data[picking_type].update(create_data[picking_type])
                 sequence = IrSequenceSudo.create(sequence_data[picking_type])
+                existing_sequence = IrSequenceSudo.search_count([('company_id', '=', sequence.company_id.id), ('name', '=', sequence.name)], limit=1)
+                if existing_sequence:
+                    sequence.name = sequence.name + " "  + "[" + str(sequence.id) + "]"
```

Comment:
```suggestion
                    sequence.name = _("%s (copy)(%s)", (sequence.name, str(sequence.id)))
```
After discussing with the team, it would be better to have something clearly identified as a copied name (and thus translatable)